internal/commands/pull/command_test.go

```diff
@@ -233,25 +251,68 @@ Successfully pulled app down: app
 		realmClient.ExportFn = func(groupID, appID string, req realm.ExportRequest) (string, *zip.Reader, error) {
 			return "app_20210101", &zipPkg.Reader, nil
 		}
-		realmClient.ExportDependenciesFn = func(groupID, appID string) (string, io.ReadCloser, error) {
+		realmClient.ExportDependenciesArchiveFn = func(groupID, appID string) (string, io.ReadCloser, error) {
 			return "node_modules.zip", depsPkg, nil
 		}
 		realmClient.CompatibleTemplatesFn = func(groupID, appID string) ([]realm.Template, error) {
 			return nil, nil
 		}
 
-		cmd := &Command{inputs{Project: "elsewhere", LocalPath: "app", IncludeDependencies: true}}
+		cmd := &Command{inputs{Project: "elsewhere", LocalPath: "app", IncludeNodeModules: true}}
```

Comment:
likewise, lets have coverage for both `IncludeNodeModules` and `IncludeDependencies` here